images/builder/Dockerfile

```
@@ -0,0 +1,34 @@
+# project aquarium's container builder
+# Copyright (C) 2021 SUSE, LLC.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+FROM registry.opensuse.org/opensuse/leap:15.3
+LABEL aquarium.image.author="Joao Eduardo Luis <joao@suse.com>"
+LABEL aquarium.url="https://aquarist-labs.io"
+LABEL aquariumg.github="https://github.com/aquarist-labs/aquarium"
+LABEL version="0.1.0"
```

```suggestion
LABEL version=$(git describe --match 'v*')
```
keep in sync with the aquarium version?

---

this isn't valid syntax, apparently :( Probably needs to go through an ARG or something, and I'm not sure it's worth adding a new argument to the build to make this work.